numpy/core/tests/test_multiarray.py

```diff
@@ -6355,50 +6360,52 @@ def test_string(self):
         assert_equal(np.where(False, b, a), "abcd")
 
 
-class TestSizeOf(TestCase):
+if not IS_PYPY:
```

Comment:
Using dec.skip() in a setup() class method did not skip the class. It seems to skip one or more tests but not all. I could not find documentation or examples of how to do this in the 10 minutes I tried.